editor.planx.uk/src/@planx/components/Notice/Public.tsx

```diff
@@ -112,6 +115,7 @@ const NoticeComponent: React.FC<Props> = (props) => {
             size="large"
             type="submit"
             onClick={props.resetPreview}
+            sx={{ ...contentFlowSpacing(theme) }}
```

Comment:
```suggestion
            sx={contentFlowSpacing}
```

This is a bit more terse, and allows us to avoid an extra import of `useTheme()`.

We can pass a callback, `functionName` which is just a shorthand for `(x) => functionName(x)`. 

The `sx` prop allows us to natively use the theme in callbacks so in this example `sx={(theme) => contentFlowSpacing(theme) }` is the same as `sx={contentFlowSpacing}`

Reply:
Thanks @DafyddLlyr, will update.